internal/backends/winit/event_loop.rs

```diff
@@ -378,6 +386,15 @@ impl EventLoopState {
                 };
                 let ev = match state {
                     winit::event::ElementState::Pressed => {
+                        if button == PointerEventButton::Left
+                            && self.current_resize_direction.is_some()
+                        {
+                            handle_resize(
+                                window.winit_window().as_ref(),
+                                self.current_resize_direction,
+                            )
```

Comment:
I believe we should also return here if the event is processed, so we don't forward the event to the window if it is on the border.